CodeSystem-England-MessageEventsBars.xml

```diff
@@ -0,0 +1,43 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
+    <id value="England-MessageEventsBars" />
+    <url value="https://fhir.nhs.uk/CodeSystem/message-events-bars" />
+    <version value="1.0.0" />
+    <name value="EnglandMessageEventsBars" />
+    <title value="England Message Events Bars" />
+    <status value="draft" />
+    <date value="2024-06-14" />
+    <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+    <description value="Common messaging events for Bars Service Requests." />
```

Comment:
Suggest: A set of codes that define common messaging events for BARS Service Requests.